doc/lua_api.txt

```diff
@@ -1535,19 +1553,22 @@ Tools define:
         * Digging times
         * Damage groups
 
-#### Full punch interval
+Full punch interval
+-------------------
 When used as a weapon, the tool will do full damage if this time is spent
 between punches. If e.g. half the time is spent, the tool will do half
 damage.
 
-#### Maximum drop level
+Maximum drop level
```

Comment:
This is actually `###` a level below `Tools definition`as it describes a part of the tool definition.